src/components/Graph.react.js

```diff
@@ -18,11 +19,14 @@ const filterEventData = (gd, eventData, event) => {
         const data = gd.data;
         for(let i=0; i < eventData.points.length; i++) {
             const fullPoint = eventData.points[i];
-            const pointData = filter(function(o) {
+            var pointData = filter(function(o) {
```

Comment:
we aren't mutating `pointData` anywhere after this, so it can remain `const` right?